lib/linter.js

```diff
@@ -178,20 +178,24 @@ var runChecks = function (ast, fullPath, lines, config) {
     return results;
 };
 
-var mergeLinters = function (defaultLinters, config) {
+var mergeLintersFromConfig = function (defaultLinters, config) {
```

Comment:
Ehh it seemed a little inaccurate to me. It wasn't merging two linter arrays; it was merging a linter array with linters and/or linter descriptions from a config.